tests/Unit/Domain/Python/Test_InertialCoordinates.py

```diff
@@ -0,0 +1,45 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from spectre.Domain import (deserialize_domain_3d,
+                            deserialize_functions_of_time,
+                            inertial_coordinates_3d, ElementId3D)
+
+import numpy as np
+import numpy.testing as npt
+import os
+import spectre.IO.H5 as spectre_h5
+import unittest
+from spectre.DataStructures import DataVector
+from spectre.Informer import unit_test_src_path
+
+
+class TestInertialCoordinates(unittest.TestCase):
+    def test_inertial_coordinates(self):
+        volfile_name = os.path.join(unit_test_src_path(),
+                                    "Visualization/Python/VolTestData0.h5")
+        with spectre_h5.H5File(volfile_name, "r") as open_h5_file:
+            volfile = open_h5_file.get_vol("/element_data")
+            obs_id = volfile.list_observation_ids()[0]
+            serialized_domain = volfile.get_domain(obs_id)
+            serialized_fot = volfile.get_functions_of_time(obs_id)
+        domain = deserialize_domain_3d(serialized_domain)
+        functions_of_time = deserialize_functions_of_time(serialized_fot)
+
+        element_id = ElementId3D("[B0,(L1I0,L0I0,L0I0)]")
+        element_logical_coords = 3 * [DataVector([-1., 1.])]
+        inertial_coords = np.asarray(
+            inertial_coordinates_3d(element_logical_coords,
+                                    element_id=element_id,
+                                    domain=domain,
+                                    time=0.,
+                                    functions_of_time=functions_of_time))
+
+        # Domain is [0, 2 pi]^3
+        npt.assert_allclose(
+            inertial_coords,
+            np.array([[0., np.pi], [0., 2 * np.pi], [0., 2 * np.pi]]))
```

Comment:
The comment and the actual array don't match.

Reply:
It's correct, extended it a bit to make it clearer :)